TeachingRecordSystem/src/TeachingRecordSystem.Core/Dqt/Models/NoteTextWithoutHtmlAsync.cs

```diff
@@ -0,0 +1,58 @@
+using System.Text;
+using AngleSharp;
+using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+
+namespace TeachingRecordSystem.Core.Dqt.Models;
+
+public static class StringExtensions
+{
+    public static async Task<string> NoteTextWithoutHtmlAsync(this string note)
```

Comment:
Can we make this a method on `DqtNote` instead? It's a very niche thing to have as an extension method over all `string`s.